composer.json

```diff
@@ -140,7 +144,9 @@
     "twig/extensions": "^1.5.1",
     "twig/twig": "^2.4.8",
     "vasek-purchart/console-errors-bundle": "^1.0.1",
-    "webmozart/assert": "^1.4"
+    "webmozart/assert": "^1.4",
+    "trikoder/oauth2-bundle": "^1.1",
+    "nyholm/psr7": "^1.1"
```

Comment:
* removed `nyholm/psr7` as it wasn't used
* added `trikoder/oauth2-bundle` to `project-base`... please check https://github.com/shopsys/shopsys/pull/1055/commits/6a8d8665018f5970c0a2ecf08eeaad92067d982f because I moved oauth route configuration to project-base (api bundle doesn't depend on oauth bundle)